packages/story-editor/src/components/form/dateTime/utils.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export function getCalenderType(weekStartsOn) {
+  let calenderType;
+  switch (weekStartsOn) {
+    case 0:
+      calenderType = 'US';
+      break;
+    case 1:
+      calenderType = 'ISO 8601';
+      break;
+    case 5:
+      calenderType = 'Hebrew';
+      break;
+    case 6:
+      calenderType = 'Arabic';
+      break;
+    default:
+      calenderType = 'ISO 8601';
+  }
+  return calenderType;
```

Comment:
This can be simplified:

```suggestion
  const calendarTypes = {
    0: 'US',
    1: 'ISO 8601',
    5: 'Hebrew',
    6: 'Arabic',
  };
  return calendarTypes[weekStartsOn];
```

Passing a value other than 0,1,5 or 6 will cause this to return `undefined`, which will cause `react-calendar` to just use `ISO 8601` as the default anyway.